api/test/context/BUILD

```diff
@@ -0,0 +1,25 @@
+load("//bazel:otel_cc_benchmark.bzl", "otel_cc_benchmark")
+
+cc_test(
+    name = "context_test",
+    srcs = [
+        "context_test.cc",
+    ],
+    deps = [
+        "//api",
+        "@com_google_googletest//:gtest_main",
+    ],
+)
+
+cc_test(
+    name = "runtimeContext_test",
+    srcs = [
+        "runtimeContext_test.cc",
```

Comment:
Please use lowercase + underscores, i.e. runtime_context_test (also on L15) to be consistent with the rest of the files.